src/views/viewer/ChannelView/ChannelView.tsx

```diff
@@ -276,12 +276,20 @@ export const ChannelView: React.FC = () => {
               setIsSearchingInputOpen={setIsSearchingInputOpen}
               setIsSearching={setIsSearching}
               search={search}
+              isSearching={isSearching}
+              setCurrentTab={setCurrentTab}
             />
           )}
-          {currentTab === 'Videos' && !isSearching && (
+          {currentTab === 'Videos' && (
```

Comment:
Previously the sorting select wasn't displayed at all when in search results because we cannot change their order. The updated design added the disabled select with `Best match` selected but the implementation doesn't reflect that

<img width="682" alt="Screenshot 2021-09-21 at 11 21 41" src="https://user-images.githubusercontent.com/12646744/134145913-338ea775-d87f-4bbf-95df-065642229e25.png">

<img width="880" alt="Screenshot 2021-09-21 at 11 19 50" src="https://user-images.githubusercontent.com/12646744/134145882-907a08bf-f64a-4ca3-8ab7-1a0eca95b60c.png">